adoc/headers/accessorBuffer.h

```diff
@@ -184,7 +184,7 @@ class accessor {
   cl::sycl::atomic<DataT, access::address_space::global_space>
   operator[](id<Dimensions> index) const;
 
-  std::add_pointer_t<value_type> get_pointer() const noexcept;
+  accessor_ptr<access::decorated::legacy> get_pointer() const noexcept;
```

Comment:
The table says this function is deprecated.  I think it would be good to add a comment saying that here too, similar to what we have above for the other deprecated functions.

Reply:
Fixed in https://github.com/KhronosGroup/SYCL-Docs/pull/432/commits/b84963dbb7a353890317fcaa01c8c172a727ae68.